synchronize.py

```diff
@@ -127,7 +127,18 @@ def create_component(self, project, branch):
         try:
             print subprocess.check_output(cmd)
         except subprocess.CalledProcessError:
-            logger.error('WeblateAPI.create_component > Error processing the project %s on branch %s' % (project['slug'], branch['branch_name']))
+          logger.error('WeblateAPI.create_component > Error processing the project %s on branch %s' % (project['slug'], branch['branch_name']))
```

Comment:
@moylop260 
Thanks for you comment
I register this PR https://github.com/Vauxoo/SynRunbotWeblate/pull/5